code-challenges.test.js

```diff
@@ -49,6 +74,19 @@ var testArray1 = [3, 7, 10, 5, 4, 3, 3]
 var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
+//create a describe function to test the array
+describe("noDuplicateValues", () => {
+    // an it/expect method, nested within the describe block, that in plain words, describes that the function does.
+    it("takes in two arrays as arguments and returns one array with no duplicate values.", () => {
+        //create expect method, nest method in test block, check that new array matches the expected output
+        expect(noDuplicateValues(testArray1,testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+    })
+})
 
+// ReferenceError: noDuplicateValues is not defined
 
 // b) Create the function that makes the test pass.
+
+function noDuplicateValues(array1, array2) {
+    return newArray =  [...new Set([...array1, ...array2])]
+}
```

Comment:
I don't see any need to have the `newArray =` section here. The rest is doing the job.